doc/cephfs/charmap.rst

```diff
@@ -0,0 +1,185 @@
+CephFS Directory Entry Name Normalization and Case Folding
+==========================================================
+
+CephFS allows configuring directory trees to **normalize** or **case fold**
+directory entry names. This is typically a useful property for file systems
+exported by gateways like Samba which enforce a case-insensitive view of the
+file system, typically with performance penalties on file systems which are not
+case-insensitive.
+
+The following virtual extended attributes control the **character mapping**
```

Comment:
Suggest no style for `character mapping` as it doesn't seem here to be a literal or a specialized term.

Reply:
I believe it is a special term in the context of this document (after all, the document is titled "charmap").